src/platform/site-wide/sass/style-consolidated.scss

```diff
@@ -30,10 +30,10 @@ $modal-layer: 400;
 $font-stack: "Source Sans Pro", "Helvetica Neue", "Helvetica", "Roboto", "Arial", sans-serif;
 
 @font-face {
-  font-family: "FontAwesome";
-  src: url("~font-awesome/fonts/fontawesome-webfont.woff2") format("woff2"),
-    url("~font-awesome/fonts/fontawesome-webfont.woff") format("woff"),
-    url("~font-awesome/fonts/fontawesome-webfont.eot") format("eot");
+  font-family: "Font Awesome 5 Free";
+  src: url("~@fortawesome/fontawesome-free/webfonts/fa-solid-900.woff2") format("woff2"),
+    url("~@fortawesome/fontawesome-free/webfonts/fa-solid-900.woff") format("woff"),
+    url("~@fortawesome/fontawesome-free/webfonts/fa-solid-900.eot") format("eot");
```

Comment:
Can we update this to import fonts from the Formation module instead? If we can do this, then I think we can uninstall Font Awesome from `vets-website` altogether, and instead rely entirely on Formation's FA.